src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {

```

Comment:
According to the checklist you need to add a private default constructor to HibernateUtil class in order to prevent the creation HibernateUtil objects.
`private HibernateUtil() {
    }`